src/libraries/System.Net.Http/tests/StressTests/HttpStress/Dockerfile

```diff
@@ -1,7 +1,8 @@
 ARG SDK_BASE_IMAGE=mcr.microsoft.com/dotnet/nightly/sdk:5.0-buster-slim
 FROM $SDK_BASE_IMAGE
 
-RUN echo "DOTNET_SDK_VERSION="$env:DOTNET_SDK_VERSION
+RUN echo "DOTNET_SDK_VERSION="$DOTNET_SDK_VERSION
```

Comment:
I think we should get rid of these prints completely in all dockerfiles.

Reply:
It was suggested to add by .Net Engineering Services team to make investigations easier.